src/trainer.py

```diff
@@ -895,14 +885,12 @@ def _validate(self, epoch, tasks, batch_size, periodic_save=True):
         Validate on all tasks and return the results and whether to save this epoch or not.
         Note/TODO: 'Epoch' here refers to validation passes, not proper epochs over
         any given task's training set.
-
```

Comment:
This isn't deleted - not sure why github is showing that it is.

Reply:
Probably out of sync with master...